adapters/Mintegral/MintegralAdapter/GADMediationAdapterMintegralConstants.h

```diff
@@ -15,7 +15,7 @@
 #import <Foundation/Foundation.h>
 
 /// Mintegral mediation adapter version.
-static NSString *const GADMAdapterMintegralVersion = @"7.4.2.0";
+static NSString *const GADMAdapterMintegralVersion = @"7.4.3.0";
```

Comment:
@Mintegral-Lee We already updated the version, can you rebase this pull request?